index.d.ts

```diff
@@ -29,10 +36,14 @@ export interface EditorInfo {
 }
 
 /**
-Open the given files in the user's editor at specific line and column if supported by the editor. It does not wait for the editor to start or quit.
+Open the given files in the user's editor at specific line and column if supported by the editor. It does not wait for the editor to start or quit  unless you specify `wait: true` in the options.
 
 @param files - Items should be in the format `foo.js:1:5` or `{file: 'foo.js', line: 1: column: 5}`.
 
+@param {boolean} [options.wait=false] - Wait until the editor is closed (default: false).
+
+@returns Promise<void> - If options.wait is true, a promise is returned which resolves as soon as the editor closes.
```

Comment:
The JS code always returns a promise, so this is incorrect.

Reply:
New wording: https://github.com/sindresorhus/open-editor/pull/17/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R43